public/app/features/alerting/unified/components/contact-points/useNotificationTemplates.ts

```diff
@@ -14,50 +23,137 @@ interface BaseAlertmanagerArgs {
 }
 
 export interface NotificationTemplate {
+  uid: string;
   name: string;
   template: string;
   provenance: string;
 }
 export function useNotificationTemplates({ alertmanager }: BaseAlertmanagerArgs) {
   const { useGetAlertmanagerConfigurationQuery } = alertmanagerApi;
+  const { useListNamespacedTemplateGroupQuery } = templatesApi;
```

Comment:
It feels cleaner to me to have all of the API destructuring once at the top of the file - what do you think?

```ts
const {
  useCreateNamespacedTemplateGroupMutation,
  useDeleteNamespacedTemplateGroupMutation,
  useLazyReadNamespacedTemplateGroupQuery,
  useReplaceNamespacedTemplateGroupMutation,
  useListNamespacedTemplateGroupQuery,
} = templatesApi;
const { useGetAlertmanagerConfigurationQuery, useLazyGetAlertmanagerConfigurationQuery } = alertmanagerApi;
```

Reply:
Yes!